sdk/core/Cargo.toml

```diff
@@ -62,7 +62,8 @@ hmac_openssl = ["dep:openssl"]
 test_e2e = []
 azurite_workaround = []
 xml = ["quick-xml"]
-tokio-fs = ["tokio/fs", "tokio/io-util"]
+tokio-fs = ["tokio/fs", "tokio/sync", "tokio/io-util"]
```

Comment:
What changed to the tokio-fs feature that requires `tokio/sync`?

Reply:
It uses mutex which is behind sync feature flag. It just didn't compile. Should fix ci tests as well